pages/Discord-Rules.md

```diff
@@ -1,7 +1,7 @@
 If you join our [Discord Server](https://discord.gg/fsD4Bkh), then we want you to follow these rules.<br>
```

Comment:
```suggestion
If you join our [Discord Server](https://discord.gg/slimefun), then we want you to follow these rules.<br>
```
better to update this to vanity link? or this is intended to count traffic from here?